bcda/cclf/cclf_test.go

```diff
@@ -34,6 +34,44 @@ func TestCCLFTestSuite(t *testing.T) {
 	suite.Run(t, new(CCLFTestSuite))
 }
 
+func (s *CCLFTestSuite) TestImportCCLFDirectory_PriorityACOs() {
+	// The order they should be ingested in. 1 and 2 are prioritized; 3 is the other ACO in the directory.
+	var aco1, aco2, aco3 = "A9989", "A9988", "A0001"
```

Comment:
I could be missing it, but do we have a test for a scenario where an ACO ID exists in our `CCLF_PRIORITY_ACO_CMS_IDS` env var but does not exist on the system?  I was thinking we should make sure that we don't bomb in that scenario, because it is definitely possible it could happen.

Reply:
This test shows that any ACO IDs in the priority list that aren't in the CCLF file metadata map will get ignored: https://github.com/CMSgov/bcda-app/pull/392/files#diff-71a7661f40e51b958b167a5dfb4fda57R489